tests/unit/materializers/test_base_materializer.py

```diff
@@ -20,6 +20,7 @@
 
 
 class TestMaterializer(BaseMaterializer):
+    __test__ = False
```

Comment:
Interesting I didn't know about this way to disable the pytest warning. I renamed the class in a separate PR to not start with `Test` to fix the issue instead. I think that's on develop already, feel free to either change my renaming or remove the `__test__`, just both is unnecessary imo